modules/config.tf

```diff
@@ -0,0 +1,42 @@
+locals {
```

Comment:
why we need /modules/xyz.tf files?
our module submodule structure have this form:
/xyz.tf
/modules/submodule-name-here/xyz.tf